contributing_to_docs/doc_guidelines.adoc

```diff
@@ -690,15 +690,17 @@ You must get approval from the Engineering, QE, and Docs teams before embedding
 
 == Indicating Technology Preview features
 
-To indicate that a feature is in Technology Preview, include the
-`snippets/technology-preview.adoc` file in the feature's assembly to keep the
-supportability wording consistent across Technology Preview features and provide a value for the :FeatureName: variable before you include this module.
+To indicate that a feature is in Technology Preview, include the `snippets/technology-preview.adoc` file in the feature's assembly or module to keep the supportability wording consistent across Technology Preview features. Provide a value for the `:FeatureName:` variable before you include this module.
 
-See link:https://github.com/openshift/openshift-docs/pull/13878/files#diff-615ba1bf3b09d11a9c2604b775aa32f2[an example] of how this is applied.
+[source,text]
+----
+:FeatureName: The XYZ plug-in
+\include::snippets/technology-preview.adoc[]
```

Comment:
Do you need the \ here?
Will you fix the typo on line 110? 
Otherwise, this LGTM!

Reply:
Yea, it escapes the include directive, otherwise it renders like:

![image](https://user-images.githubusercontent.com/3442316/152611822-68e8f92e-4a83-42ee-8a5b-29d1bf79fdd8.png)

Will fix the line 110 ordered numbering too. :+1: